### DISCUSSION TITLE
Node version

### DISCUSSION THREAD

**Discussion by fjr619** [varabyte/kobweb#367] [Q&A]
```
jetbrains-compose = "1.5.1"
kobweb = "0.14.2"
kotlin = "1.9.10"
```

I have my node install in `C:\Program Files\nodejs` (my version 18.18.2)

But when i tried to make project kobweb, it will download node again version 18.12.1, and it will save to `C:\Users\Admin\.gradle\nodejs`

how to pointing that to my node in program files? so not using node from .gradle
fyi i'm using Android Studio to compile kobweb project

![image](https://github.com/varabyte/kobweb/assets/9444636/b5f05a83-c7c7-4bef-9bc7-552a8b606895)
![image](https://github.com/varabyte/kobweb/assets/9444636/c8b3d38b-1c63-4389-b22d-5e2b8852c483)

**Comment by bitspittle** [comment]
I'm not an expert here. All the node stuff is handled automatically by the JetBrains Multiplatform Compose Gradle plugin.

Still, check out this code snippet I found:

https://github.com/JetBrains/compose-multiplatform/blob/d777238d51c767f4e8bacb5b37c7d6b60043176c/examples/html/compose-in-js/build.gradle.kts#L46

Maybe you can try setting it to 18.12.1 in your own project? And see what other fields come up via autocomplete?
